### PR TITLE
BrightId flow (UI, not functionality)

### PR DESCRIPTION
**Pull request by ryancreatescopy**
**NOTE**: for local dev, configure `VUE_APP_USER_REGISTRY_TYPE=brightid`

- [x] @samajammin to get to a state where we can merge this PR w/ UI implemented
- [ ] @samajammin create issues for remaining TODOs as BrightID epic

https://www.notion.so/efdn/Set-up-BrightID-flow-UI-and-routing-50-af0b4087079844719214f05e119a6ef9
- [x] Create routes for registering with BrightId
- [ ] Check if user registry is full before allowing user to go through verification - close out if full (just like /join flow)
- [ ] Check if contribution phase is over before allowing user to go through verification - close out if past (just like /join flow)
- [ ] Share flow with BrightID team for feedback

**Status:** The flow UI is pretty much done, just needs review. 

**Todos, but maybe for other PRs:**

Plugging in actual logic to components
- [ ] add logic to check status and ensure verification component reflects status
  - [ ] Confirm we can fetch if user registry is full / filling up - may need to add view function to registry contract
- [ ] logic to move users through the process (+ display toast messages) if user has already completed certain steps by virtue of using clr or gitcoin before: (see different states here: https://www.figma.com/file/gnA82CTLGjk0TfFHMvgyrK/?node-id=595%3A6310)
- [ ] ensure "pending" loader on verification step displays as pending while user is unverified but has started the process (e.g. looked at the screen)  - see Figma screen: https://www.figma.com/file/gnA82CTLGjk0TfFHMvgyrK/CLR?node-id=1652%3A5810
- [ ] Check for any other hardcoded logic :)

Update content
- [ ] update transaction widget with associated success/error states (see #60)
- [ ] add info about ways to verify (atm it's just a shell, it needs updating to include links to BrightID zoom party schedule, our event, etc.) on /setup/get-verified/verification
- [ ] add external links for google meet / learn to /setup landing page
- [ ] ways to verify accordion needs fixing (currently all accordions open/close at same time) - create an accordion component for this

### CODE REVIEW

**Comment by ryancreatescopy** [comment]
T 👏 S 👏  L 👏 I 👏 N 👏 T 👏 I 👏 N👏 G

File: vue-app/src/router/index.ts [hunk 1]
```diff
@@ -66,6 +69,21 @@ const routes = [
     name: 'join',
     component: JoinLanding,
   },
+  {
+    path: '/setup',
+    name: 'setup',
+    component: SetupLanding,
+  },
+  {
+    path: '/setup/get-verified/:step',
```
Comment (samajammin) [inline vue-app/src/router/index.ts]: @ryancreatescopy curious on your reasoning for /get-verified/ subroute - is the thinking there will be other /setup/ routes, e.g. for l2?

**Comment by ryancreatescopy** [comment]
Yep, exactly that.

**Comment by ryancreatescopy** [comment]
re: previous commit. More info: https://www.notion.so/efdn/CLR-help-articles-58e3630d9fa045e39f74cadb51e74009

**Comment by ryancreatescopy** [comment]
Updated description with status/progress

**Review by samajammin — APPROVED** [review]
Merging without addressing TODOs in PR description - will create separate issue for those.

**Pull request merged**